pkg/rekor/identity.go

```diff
@@ -94,8 +107,22 @@ func (e *IdentityEntry) String() string {
 	return strings.Join(parts, " ")
 }
 
+// ConcatOIDMatchers groups all OID matchers from OIDMatchers, FulcioExtensions, and CustomOIDs into one slice
+func ConcatOIDMatchers(mvs MonitoredValues) ([]OIDMatcher, error) {
```

Comment:
This function can be lowercase since it doesn’t need to be exported outside the package.